deepTumour.wdl

```diff
@@ -68,6 +68,7 @@ task runDeepTumour {
         set -euo pipefail
         
         mkdir out
+        source $DEEP_TUMOUR_ROOT/.venv/bin/activate
         python $DEEP_TUMOUR_ROOT/src/DeepTumour.py --vcfFile ~{vcf} --reference $HG19_ROOT/hg19_random.fa --~{liftover} --outDir out
```

Comment:
You may use ~{"--" + liftover} instead, may be a cleaner solution as at present it would have "--" passed to script when liftover is not needed. When you evaluate ~{"--" + liftover} it is going to evaluate to empty if no liftover is requested

Reply:
Thank Peter, I changed the String liftover generating line to:
`String liftover = if reference_genome == "hg38" then "--hg38" else ""`
to fix the issue. Also added a rgression test with hg19 reference that does not need liftover.